source/common/res/features/pacing/main.js

```diff
@@ -123,34 +149,32 @@
-                  if (pace > 1) {
-                    temperature = 'cautious';
-                  } else {
-                    temperature = 'positive';
-                  }
+                  const showIndicator = ynabToolKit.options.pacing === '2';
+                  const showDays = ynabToolKit.options.pacing === '3';
 
                   var deemphasized = masterCategory.get('isDebtPaymentCategory') || $.inArray(masterCategoryDisplayName + '_' + subCategoryDisplayName, deemphasizedCategories) >= 0;
                   var display = Math.round((budgeted * timeSpent() - activity) * 1000);
-                  var tooltip;
-
-                  if (display >= 0) {
-                    tooltip = 'In ' + transactionCount + ' transaction' + (transactionCount !== 1 ? 's' : '') +
-                      ' you have spent ' + ynabToolKit.shared.formatCurrency(display, false) +
-                      ' less than your available budget for this category ' + Math.round(timeSpent() * 100) +
-                      '% of the way through the month.&#13;&#13;' + (deemphasized ? 'Click to unhide.' : 'Click to hide.');
-                  } else if (display < 0) {
-                    tooltip = 'In ' + transactionCount + ' transaction' + (transactionCount !== 1 ? 's' : '') +
-                      ' you have spent ' + ynabToolKit.shared.formatCurrency(-display, false) +
-                      ' more than your available budget for this category ' + Math.round(timeSpent() * 100) +
-                      '% of the way through the month.&#13;&#13;' + (deemphasized ? 'Click to unhide.' : 'Click to hide.');
-                  }
-
-                  $(this).append('<li class="budget-table-cell-available budget-table-cell-pacing"><span title="' + tooltip +
-                               '" class="budget-table-cell-pacing-display currency ' + temperature +
-                               (deemphasized ? ' deemphasized' : '') + (showIndicator ? ' indicator' : '') +
-                               '" data-name="' + masterCategoryDisplayName + '_' + subCategoryDisplayName + '">' +
-                               ynabToolKit.shared.formatCurrency(display, true) + '</span></li>');
+                  const displayInDays = getDaysAheadOfSchedule(display, budgeted, activity);
+
+                  const days = Math.abs(displayInDays) === 1 ? 'day' : 'days';
+                  const formattedDisplay = showDays ? `${displayInDays} ${days}`
+                    : ynabToolKit.shared.formatCurrency(display, true);
+
+                  const tooltip = getTooltip(display, displayInDays, transactionCount, deemphasized);
+                  const deemphasizedClass = deemphasized ? 'deemphasized' : '';
+                  const indicatorClass = showIndicator ? 'indicator' : '';
+                  const temperatureClass = (pace > 1) ? 'cautious' : 'positive';
+                  $(this).append(`
+                    <li class="budget-table-cell-available budget-table-cell-pacing">
```

Comment:
Doing HTML this way may get us rejected by Firefox review, but I'll go ahead and give it a whirl and see what happens.